zbx_redis_stats.py

```diff
@@ -97,9 +97,8 @@ def llen():
                 print(client.llen(args.db))
 
             def llensum():
-                keys = client.keys('*')
```

Comment:
It's better set
```python
keys = client.scan_iter('*')
```
and use `keys` instead of use:
```python
for key in client.scan_iter('*'):
```
because it will call `client.scan_iter('*')` just once, and not for each `for`-loop iteration.